services/getstream/activityFormatter.js

```diff
@@ -80,7 +80,7 @@ const activityFormatter = async (item, feedGroup, userId, excludedPostParameter)
   }
 
   let newItem = item;
-  newItem = modifyReactionsPost(newItem, newItem.anonimity);
+  newItem = modifyReactionsPost(newItem);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyReactionsPost` function is now being called with only one argument, whereas previously it was called with two. This change could potentially cause issues if the `modifyReactionsPost` function still expects two arguments. Please ensure that the function definition of `modifyReactionsPost` has been updated to handle this change correctly.
```diff
- newItem = modifyReactionsPost(newItem, newItem.anonimity);
+ newItem = modifyReactionsPost(newItem);
```
If the function has not been updated, you might need to reconsider this change or update the function definition accordingly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->